terraform/modules/ecs_module/alb.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_alb" "main" {
```

Comment:
🚫 **[tfsec]** <[AVD-AWS-0053](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/elb/alb-not-public/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Load balancer is exposed publicly.